chrome-extension/popup.js

```diff
@@ -161,7 +236,7 @@ function logout() {
 function setAccountInfo() {
   if(loggedIn) {
     gapi.client.request({
-      path: USER_INFO_URL + accesToken
+      path: USER_INFO_URL + accessToken
     }).then(response => {
```

Comment:
Should we be requesting this information from our own `user-registration` servlet on a GET request now that we have that?

Reply:
You're right! I'll change that